### PR TITLE
feat(build): Support sentry webpack plugin

### LINKED ISSUE #79: Support sentryWebpackPlugin
It would be good to support sentryWebpackPlugin in the app-builder

### PR DESCRIPTION
**Pull request by n-parpibaeva**
Closes #79

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -223,6 +223,7 @@ With this `{rootDir}/src/ui/tsconfig.json`:
   - `endpoint` (`string`) - cdn host to upload files
   - `compress` (`boolean`) - upload also gzip and brotli compressed versions of files
   - `additionalPattern` (`string[]`) — patterns for uploading additional files. By default, only files generated by webpack are loaded.
+- `sentryConfig` (`Options`) — `@sentry/webpack-plugin` [configuration options](https://www.npmjs.com/package/@sentry/webpack-plugin). Requires `hiddenSourceMap: false` and `disableSourceMapGeneration: false`.
```
Comment (ValeraS) [inline README.md:226]: There is no documentation
image

**Review by ValeraS — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (n-parpibaeva) [reply README.md:226]: It opens from time to time, I can replace it with https://github.com/getsentry/sentry-javascript-bundler-plugins/blob/main/packages/webpack-plugin/README_TEMPLATE.md#example

[the remark this answers is not in the document]
Comment (ValeraS) [reply README.md:226]: `hidden-source-map` does not disable source map generation

File: src/common/models/index.ts [hunk 2]
```diff
@@ -14,6 +14,7 @@ import type {Options as CircularDependenciesOptions} from 'circular-dependency-p
 import type {Config as SvgrConfig} from '@svgr/core';
 import type {ForkTsCheckerWebpackPluginOptions} from 'fork-ts-checker-webpack-plugin/lib/plugin-options';
 import type {Options as StatoscopeOptions} from '@statoscope/webpack-plugin';
+import {SentryWebpackPluginOptions} from '@sentry/webpack-plugin';
```
[the remark this answers is not in the document]
Comment (n-parpibaeva) [reply src/common/models/index.ts:17]: it would be nice if a linter highlighted this

**Review by ValeraS — APPROVED** [review]

**Pull request merged**